src/init.jsx

```diff
@@ -7,6 +7,7 @@ import { ipcRenderer } from 'electron';
 
 import Application from './components/Application/Application.jsx';
 import socketInit, { socketMessageHook } from './socket.js';
+import { logger, notifications } from './store/middleware.js';
```

Comment:
Is middleware a redux store?

Reply:
no, but it is implemented by the store, so to speak. I don't think there's any good reason to create a folder for it (if you were considering it), when

a - the term middleware is easily confused with other things (esp. express / http servers etc)
b - we can contain all our middleware in one file for now; unless it should grow.